spec/rubocop/cop/lint/empty_when_spec.rb

```diff
@@ -172,6 +172,29 @@
         end
       RUBY
     end
+
+    it 'registers an offense for missing when body without a comment' do
+      expect_offense(<<~RUBY)
+        case condition
+        when foo
+        42 # magic number
+        when bar
+        ^^^^^^^^ Avoid `when` branches without a body.
+        when baz # more comments mixed
+        21 # another magic number
+        end
```

Comment:
Here's my two cents. It looks better to indent.

```suggestion
        case condition
        when foo
          42 # magic number
        when bar
        ^^^^^^^^ Avoid `when` branches without a body.
        when baz # more comments mixed
          21 # another magic number
        end
```